### PR TITLE
Change repo resource to contain wildcard

### PR DESCRIPTION
**Pull request by guy-har**

### CODE REVIEW

**Review by arielshaqed — APPROVED** [review]
Thanks!  I must have dropped this change somewhere on my stack of PRs... good catch. :-(

**Comment by guy-har** [comment]
Add some more fixes on this PR
Every commit fixes a specific issue

File: webui/src/pages/repositories/repository/settings/general.jsx [hunk 1]
```diff
@@ -105,10 +105,11 @@ const SettingsContainer = () => {
                 repo={repo}
                 onCancel={() => { setShowDeleteModal(false) }}
                 onSubmit={() => {
-                    repositories.delete(repo.id).catch(err => {
-                        setDeletionError(err)
-                    }).then(() => {
+                    repositories.delete(repo.id).then(() => {
```
Comment (arielshaqed) [inline webui/src/pages/repositories/repository/settings/general.jsx:108]: This change seems unrelated.  Can you explain what it does?

File: pkg/auth/acl/permission.go [hunk 2]
```diff
@@ -59,12 +59,16 @@ func ACLToStatement(acl model.ACL) (model.Statements, error) {
 		if err != nil {
 			return nil, fmt.Errorf("%s: %w", acl.Permission, ErrBadACLPermission)
 		}
+		readConfigStatement, err := auth.MakeStatementForPolicyType("FSReadConfig", all)
+		if err != nil {
+			return nil, fmt.Errorf("%s: %w", acl.Permission, ErrBadACLPermission)
+		}
```
Comment (arielshaqed) [inline pkg/auth/acl/permission.go:65]: I don't understand why we need this code.  ["FSRead" includes `fs:Read*`](https://github.com/treeverse/lakeFS/pull/5607/files#diff-1bfeff83ad028c5f61389248471de20c1d5708ca3a39c1c3e26f9d8970c043cfR59), why add ReadConfig explicitly?  (Is it because we now add `repositories/*` to the ARNs of FSRead?)

**Review by arielshaqed — APPROVED** [review]
Thanks!

Did we add repositories/* to RepositoriesToARNs in a separate PR?

File: webui/src/pages/repositories/repository/settings/general.jsx (hunk 1 above)
Comment (guy-har) [reply webui/src/pages/repositories/repository/settings/general.jsx:108]: Without this change then is called in case of catch
With the previous implementation, if the user isn't authorized he gets an error when trying to delete the repository but the `then` is called anyway and the user is redirected to the repositories page (which in some cases the user doesn't have permissions to view that page)

File: pkg/auth/acl/permission.go (hunk 2 above)
Comment (guy-har) [reply pkg/auth/acl/permission.go:65]: It's added explicitly because ReadStorageConfig requires ReadConfig on `*` which fails on when scoping read to specific repositories

**Comment by guy-har** [comment]
> Did we add repositories/* to RepositoriesToARNs in a separate PR?

Changed the fix, accidentally committed part of the change, fixed it now. 
Thanks!
